Python/Canny edge detector/canny edge.py

```diff
@@ -0,0 +1,13 @@
+import cv2
+import numpy as np
+from matplotlib import pyplot as plt
+
+img = cv2.imread('messi.jpg',0)

```

Comment:
Use Dynamic Input instead of specifying an Image

Reply:
Okay, I will do it asap.